### DISCUSSION TITLE
Rabbitmq operators on same node

### DISCUSSION THREAD

**Discussion by paywagh11** [rabbitmq/rabbitmq-server#9120] [Questions]
I have RMW cluster kubernetes operator and messaging topology operator in my cluster which are deployed in different nodes because of nodes availability.

I would like to understand whether there will be any issue related to operators operation for Rabbitmq working in case both the operators got deployed in same node?

This condition may arise in case we don't have any available extra node and only those where operator or rmq cluster is deployed.

**Comment by michaelklishin** [comment]
Both can be deployed to the same node. I don't see how they would conflict in such a case.
